scio-hdfs/src/main/scala/com/spotify/scio/hdfs/package.scala

```diff
@@ -150,6 +150,27 @@ package object hdfs {
     }
   }
 
+  /** Enhanced version of [[com.spotify.scio.io.Taps]] with HDFS methods. */
+  // TODO: scala 2.11
+  // implicit class HdfsTaps(private val self: Taps) extends AnyVal {
+  implicit class HdfsTaps(val self: Taps) {
+
+    /** Get a `Future[Tap[T]]` for a text file on HDFS. */
+    def hdfsTextFile(path: String): Future[Tap[String]] =
+      self.mkTap(s"Text: $path", () => isPathDone(path), () => HdfsTextTap(path))
+
+    /** Get a `Future[Tap[T]]` for a Avro file on HDFS. */
+    def hdfsAvroFile[T: ClassTag](path: String, schema: Schema = null): Future[Tap[T]] =
+      self.mkTap(s"Avro: $path", () => isPathDone(path), () => HdfsAvroTap(path, schema))
+
+    private def isPathDone(path: String): Boolean = {
+      val job = Job.getInstance()
```

Comment:
Is this job here created only to get configuration object out of it? then `new Configuration()` works too